mailbox/webmin_menu.pl

```diff
@@ -97,11 +97,14 @@ sub list_webmin_menu
 push(@rv, { 'type' => 'hr' });
 
 # Search box
+my $left_search_text = $text{'left_search'};
```

Comment:
Why not just change the text string rather than editing it here?

Reply:
I thought of it but it's over 40 files/languages to change.

Reply:
Fixed with using translations.